cdap-app-templates/cdap-etl/cdap-etl-realtime/src/test/java/co/cask/cdap/template/etl/realtime/RealtimeElasticsearchSinkTest.java

```diff
@@ -138,7 +138,7 @@ public void testESSink() throws Exception {
       AdapterManager adapterManager = createAdapter(adapterId, adapterConfig);
 
       adapterManager.start();
-      TimeUnit.SECONDS.sleep(5);
+      TimeUnit.SECONDS.sleep(10);
```

Comment:
@gokulavasan  Just thought of a way to do it. Spoke with @ataylor4. Fix should be coming soon.